kubernetes/zenko/templates/cosmos-operator/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "cosmos-operator.fullname" . }}
+  labels:
+    app: {{ template "cosmos-operator.name" . }}
+    chart: {{ template "zenko.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups: [""]
+  resources:
+  - persistentvolumeclaims
+  - persistentvolumes
+  verbs:
+  - "*"
+- apiGroups: ["storage.k8s.io"]
+  resources: ["storageclasses"]
+  verbs:
+  - "*"
```

Comment:
This 'operator' creates StorageClasses and such?!

Reply:
Seems like the only way... Any suggestions?

Reply:
Or well I guess we could ask for the user to provide a StorageClass name to use.

Reply:
or install a `StorageClass` as part of Zenko

Reply:
What do you think? @NicolasT